docs/src/parameter_distributions.md

```diff
@@ -16,9 +16,14 @@ Solution: We should use a Normal distribution with the predefined "bounded" cons
 
 Let's initialize the constraint first,
 ```julia
-constraint = [bounded(0,1)] # Sets up a logit-transformation into [0,1].
+constraint = [bounded(0,1)] 
 ```
-The prior is around 0.7, and the push forward of a normal distribution N(mean=1,sd=0.5) gives a prior with 95% of it's mass between [0.5,0.88].
+This sets up a logit transformation to and from the constrained space
```

Comment:
I would maybe underline that this happens automatically and the user does not have to define those functions by themselves?